### ISSUE TITLE
operation is null

### ISSUE THREAD

**Issue by allthesignals** [NullVoxPopuli/ember-query-params-service#5]
Hi there,

I tried to install (`yarn add ember-query-params-service`) and import something (`import { queryParams } from 'ember-query-params-service';`) but I'm getting `operation is null` and `Error: Could not find module 'ember-query-params-service' imported from `labs-zap-search/components/projects-map-data`.

package.json:
```
    "ember-query-params-service": "github:NullVoxPopuli/ember-query-params-service",
    "ember-source": "~3.3.1",
```

**Comment by mattmarcum** [comment]
I ran into this too.  I'm seeing the 'vendor/ember-query-params-service.js' in my chrome dev tools but none of the other code getting pulled in.  Using the latest octane-blueprint which in on ember-cli@3.9.beta2 and ember-source@3.11.canary.  It looks like this addon was created with the module-unification layout - do we need to be passing special flags or configs to ember-cli in order to get this to build correctly?

**Comment by jelhan** [comment]
This is fixed by #6, isn't it?

**Comment by NullVoxPopuli** [comment]
Closing for now -- if this is still an issue, lemme know!
